.github/workflows/pull-request.yml

```
@@ -0,0 +1,41 @@
+        id: pnpm-cache
+        run: |
+          echo "pnpm_cache_dir=$(pnpm store path)" >> $GITHUB_OUTPUT
+      - name: Setup pnpm cache
+        uses: actions/cache@v3
+        with:
+          path: ${{ steps.pnpm-cache.outputs.pnpm_cache_dir }}
+          key: ${{ runner.os }}-pnpm-store-${{ hashFiles('**/pnpm-lock.yaml') }}
+          restore-keys: |
+            ${{ runner.os }}-pnpm-store-
+      - name: Install dependencies
+        if: steps.cache.outputs.cache-hit != 'true'
```

actionlint (1.7.4)

property "cache" is not defined in object type {pnpm-cache: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)